crates/rattler/src/install/installer/mod.rs

```diff
@@ -50,6 +51,7 @@ pub struct Installer {
     target_platform: Option<Platform>,
     apple_code_sign_behavior: AppleCodeSignBehavior,
     alternative_target_prefix: Option<PathBuf>,
+    reinstall_packages: Option<Vec<PackageName>>,
```

Comment:
Maybe this should be a hashset set we only test for inclusion.

Reply:
Do you want to pass around the `HashSet`? If so we should get rid of the iterator as well in the `Transaction` I think :)

Reply:
Yeah makes sense!

Reply:
Ok I've changed it to accept and pass `HashSet`'s around :)